### PR TITLE
docs(Foundation/Theme): update documentation template

### PR DESCRIPTION
**Pull request by toncabral88**
## Description
https://jirasoftware.catho.com.br/browse/QTM-357

## Setup
to view the components behavior, run `yarn storybook`

## Review guide
- [ ] Documentation (yarn storybook)
- [ ] Code review

### CODE REVIEW

File: stories/Theme/Theme.story.mdx [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+## Quantum Theme
+
+The theme has properties which, some of them, are used in components.
+It's structured with these ones:
+
+- [breakpoints](#breakpoints)
+- [colors](#colors)
+- [gutter](#gutter)
+- [baseFontSize](#baseFontSize)
```
Comment (MarcosViniciusPC) [inline stories/Theme/Theme.story.mdx:33]: I suggest checking this reference

**Review by MarcosViniciusPC — APPROVED** [review]
[x] Documentation (yarn storybook)
[x] Code review

File: stories/Theme/Theme.story.mdx (hunk 1 above)
Comment (toncabral88) [reply stories/Theme/Theme.story.mdx:33]: Good point, but in this case you can request change because this causes a bug.

Comment (MarcosViniciusPC) [reply stories/Theme/Theme.story.mdx]: Right!

**Review by rapahaeru — DISMISSED** [review]

**Review by rapahaeru — APPROVED** [review]

**Pull request merged**